datasets/tabmwp/prepare.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
```

Comment:
Please add more calls to this script inside dataseta/prepare_all.sh as I assume there are some subsets other than test that need to be loaded?